ci/kind/kind-setup.sh

```diff
@@ -113,8 +118,6 @@ function configure_networks {
 
   # remove old networks
   nodes="$(kind get nodes --name $CLUSTER_NAME | grep worker)"
-  node_cnt=$(kind get nodes --name $CLUSTER_NAME | grep worker | wc -l)
-  nodes=$(echo $nodes)
```

Comment:
this case shouldn't be removed as it's also used in the following log, otherwise the log would look:
```
removing worker nodes kind-worker
kind-worker2 from networks
```

Reply:
Ok. Restored this one.